src/components/TricotsActual.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from "react";
+import {Helmet} from "react-helmet";
+import {connect} from "react-redux";
+import * as maillotActions from "../actions/maillotActions";
+import store from "../store";
+import SingleTrikotActual from "./SingleTrikotActual";
+import {Header, List, Image} from "semantic-ui-react";
+import * as riderStageConnectionsActions from "../actions/riderStageConnectionsActions";
+
+
+class TricotsActual extends Component {
+
+    constructor(props){
+        super(props);
+
+        this.state = {
+            updated: false
+        }
+    }
+
+    fetchCurrentMaillots(id) {
+        store.dispatch(maillotActions.getCurrentMaillots(id));
+        store.dispatch(riderStageConnectionsActions.getRiderStageConnectionsFromAPI(id));
+        this.setState({updated: true});
+    }
+
+
+    render() {
+        const {maillots} = this.props;
+        const {cons}  = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "cons" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'cons' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)